docs/events/indicators-of-compromise.asciidoc

```diff
@@ -0,0 +1,64 @@
+
+[role="screenshot"]
+image::images/indicators-table.png[Shows the Indicators table on the Intelligence page]
+
+[discrete]
+[[ti-indicators]]
+== Threat intelligence and indicators
+If you are unfamiliar with threat intelligence, think of it as a research function that helps an organization make informed decisions when taking actions to protect itself. An organization's security operation center (SOC) team is usually responsible for managing threat intelligence. A SOC team's goals are to understand current and emerging threats so they can recommend actions that will prevent, stop, and mitigate threats to an organization.
+
+An indicator, also referred to as an Indicator of Compromise (IoC), is a piece of information that represents a known malicious threat or reported vulnerability. There are many types of indicators, including URLs, files, domains, email addresses and more. Within SOC teams, threat intelligence analysts use indicators to detect, assess, and respond to threats.
+
+[discrete]
+[[setup-intelligence-page]]
+== Set up the Intelligence page
+
+Install threat intelligence integrations to add indicators to the Intelligence page.
+
+IMPORTANT: Before enabling a threat intelligence feed, ensure you have installed a {fleet-guide}/install-fleet-managed-elastic-agent.html[{fleet}-managed {agent}] on the hosts you want to monitor.
+
+. Choose one of the following:
+.. From the {security-app} main menu, click the Intelligence page, then click *Add Integrations*.
+.. From the {kib} main menu and click *Add integrations*. Scroll down list of integration categories and select *Threat Intelligence* to filter for threat intelligence integrations.
+. Select a threat intelligence integration, then complete the installation steps.
++
+NOTE: Refer to the Threat intelligence section of the https://docs.elastic.co/integrations[Elastic integration documentation] for more information about required field mappings.
+
+. Return to the Intelligence page in {elastic-sec}. Refresh the page if indicator data isn't displaying.
+
+[discrete]
+[[intelligence-page-ui]]
+== Intelligence page UI
+
+After you add indicators to the Intelligence page, you can <<examine-indicator-details,examine indicators>> and search, filter, and take actions on indicator data. Indicators are also visualized in the Trend view, which shows the occurrence of indicators over time.
+
+[role="screenshot"]
+image::images/interact-with-indicators-table.gif[width=90%][height=90%][Shows how to interact with the Intelligence page]
+
+[discrete]
+[[examine-indicator-details]]
+=== Examine indicator details
+Learn more about an indicator by clicking the *View details* button and opening the Indicators details flyout. The flyout contains these informational tabs:
+
+* *Overview*: Offers a summary of the indicator. Provides the indicator's name, the threat intelligence feed it came from, the indicator type, and additional relevant data.
```

Comment:
```suggestion
* *Overview*: Summarizes the indicator. Displays the indicator's name, the threat intelligence feed it came from, the indicator type, and additional relevant data.
```

Reply:
To match the form of the other bullet items, the first sentence should actually be verb-less. After that, no preference on what verb the next sentence uses.
```suggestion
* *Overview*: A summary of the indicator. Provides the indicator's name, the threat intelligence feed it came from, the indicator type, and additional relevant data.
```